src/assets/scripts/client/navigationLibrary/StandardRoute/StandardRouteWaypointModel.js

```diff
@@ -280,12 +281,23 @@ export default class StandardRouteWaypointModel extends BaseModel {
      * Find the matching fix from the `FixCollection` and clone its `StaticPositionModel` this `_positionModel`
      *
      * @for StandardRouteWaypointModel
-     * @method _clonePoisitonFromFix
+     * @method clonePositionFromFix
      * @param fixCollection {FixCollection}
      * @private
      */
-    clonePoisitonFromFix() {
-        const fixModel = FixCollection.findFixByName(this.name);
+    clonePositionFromFix() {
+        if (this.name[0] === VECTOR_WAYPOINT_PREFIX) {
```

Comment:
ahh.  well, you are correct there.  However, when checking for an individual character in a string, one typically uses [indexOf](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/indexOf).  Otherwise another developer might make the same assumption I did.

```javascript
// instead of
if (this.name[0] === VECTOR_WAYPOINT_PREFIX) { }

// preferred
if (this.name.indexOf(VECTOR_WAYPOINT_PREFIX) !== -1) { }
```